### PR TITLE
api: fix the CPU stats reported

### PR DESCRIPTION
**Pull request by giuseppe**
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by marusak** [comment]
It seems not to report `percpu_usage` anymore (which is probably correct as I am on CGroupsV2).
But it still does not show `system_cpu_usage` without which it is not possible to get any useful data from this.

File: pkg/api/handlers/generic/containers_stats.go [hunk 1]
```diff
@@ -67,9 +67,9 @@ func StatsContainer(w http.ResponseWriter, r *http.Request) {
...
 			CPUUsage: docker.CPUUsage{
 				TotalUsage:        stats.CPUNano,
-				PercpuUsage:       []uint64{uint64(stats.CPU)},
-				UsageInKernelmode: 0,
-				UsageInUsermode:   0,
+				PercpuUsage:       stats.PerCPU,
+				UsageInKernelmode: stats.CPUSystemNano,
+				UsageInUsermode:   stats.CPUNano - stats.CPUSystemNano,
 			},
 			SystemUsage:    0,
```
Comment (marusak) [inline pkg/api/handlers/generic/containers_stats.go]: I think this is what I crave for.

Comment (giuseppe) [reply pkg/api/handlers/generic/containers_stats.go]: added a new patch to address it

**Comment by TomSweeneyRedHat** [comment]
LGTM but it looks like our CI is still fighting.

**Comment by giuseppe** [comment]
is `rdoproject.org/github-check` blocking?

Given how often it fails, should we make it optional?

**Comment by rhatdan** [comment]
It is not blocking. Although I wish someone from RDO would look into what is failing?

**Comment by rhatdan** [comment]
@ssbarnea PTAL

**Comment by ssbarnea** [comment]
You can safely ignore the RHEL8-packaging issue, is unrelated and we already made it non-voting, until someone is fixing it.

**Pull request merged**